tutorials/v7/ntuple/ntpl003_lhcbOpenData.C

```diff
@@ -108,11 +109,44 @@ void ntpl003_lhcbOpenData()
    TH1F h("h", "B Flight Distance", 200, 0, 140);
    h.SetFillColor(48);
 
+   auto t1 = std::chrono::high_resolution_clock::now();
```

Comment:
Let's create a new, clean tutorial for chaining based on the existing H1 tutorials.  We can discuss this.